tests/test_state.cc

```diff
@@ -127,304 +128,212 @@ void doTest(core::State& s) {
     doTest(state);
-    std::cout << "test pass: havannah5pieExt" << std::endl;
   }
 
-  {
-    std::cout << "testing: havannah8pieExt" << std::endl;
+  TEST(StateTestsGroup, testhavannah8pieExt) {
     auto state = Havannah::State<8, true, true>(seed);
     doTest(state);
-    std::cout << "test pass: havannah8pieExt" << std::endl;
   }
 
-  {
-    std::cout << "testing: havannah5pie" << std::endl;
+  TEST(StateTestsGroup, testhavannah5pie) {
     auto state = Havannah::State<5, true, false>(seed);
     doTest(state);
-    std::cout << "test pass: havannah5pie" << std::endl;
   }
 
-  {
-    std::cout << "testing: havannah8pie" << std::endl;
+  TEST(StateTestsGroup, testhavannah8pie) {
     auto state = Havannah::State<8, true, false>(seed);
     doTest(state);
-    std::cout << "test pass: havannah8pie" << std::endl;
   }
 
-  {
-    std::cout << "testing: havannah5" << std::endl;
+  TEST(StateTestsGroup, testhavannah5) {
     auto state = Havannah::State<5, false, false>(seed);
     doTest(state);
-    std::cout << "test pass: havannah5" << std::endl;
   }
 
-  {
-    std::cout << "testing: havannah8" << std::endl;
+  TEST(StateTestsGroup, testhavannah8) {
     auto state = Havannah::State<8, false, false>(seed);
     doTest(state);
-    std::cout << "test pass: havannah8" << std::endl;
   }
 
-  {
-    std::cout << "testing: Outer Open Gomoku" << std::endl;
+  TEST(StateTestsGroup, testOuterOpenGomoku) {
     auto state = StateForOOGomoku(seed);
     doTest(state);
-    std::cout << "test pass: Outer Open Gomoku" << std::endl;
   }
 
-  {
-    std::cout << "testing: Mastermind" << std::endl;
+  TEST(StateTestsGroup, testMastermind) {
     auto state = Mastermind::State<10, 7, 2>(seed);
     doTest(state);
-    std::cout << "test pass: Mastermind" << std::endl;
   }
-  // Minesweeper tests fail
-  /* {
-    std::cout << "testing: Minesweeper beginner" << std::endl;
+
+  TEST(StateTestsGroup, DISABLED_testMinesweeperBeginner) {
     auto state = Minesweeper::State<8, 8, 10>(seed);
     doTest(state);
-    std::cout << "test pass: Minesweeper beginner" << std::endl;
   }
 
   // win rates for intermediate and expert are too low
   // when taking random actions
-  {
-    std::cout << "testing: Minesweeper intermediate" << std::endl;
+  TEST(StateTestsGroup, DISABLED_testMinesweeperIntermediate) {
     auto state = Minesweeper::State<15, 13, 40>(seed);
     doTest(state);
-    std::cout << "test pass: Minesweeper intermediate" << std::endl;
   }
 
-  {
-    std::cout << "testing: Minesweeper expert" << std::endl;
+  TEST(StateTestsGroup, DISABLED_testMinesweeperExpert) {
     auto state = Minesweeper::State<30, 16, 99>(seed);
     doTest(state);
-    std::cout << "test pass: Minesweeper expert" << std::endl;
   }
-  */
 
-  {
-    std::cout << "testing: Outer Open Gomoku" << std::endl;
-    auto state = StateForOOGomoku(seed);
-    doTest(state);
-    std::cout << "test pass: Outer Open Gomoku" << std::endl;
-  }
```

Comment:
What happened to this one?

Reply:
It's duplicated above I think. So I deleted the 2nd duplicate.

Reply:
looks like it!